### ISSUE TITLE
`Select Media Location` <Parent Directory> icon

### ISSUE THREAD

**Issue by wedebe** [openatv/enigma2#2851]
**Is your feature request related to a problem? Please describe.**
As a special-case "go up a level" entry in the file list, <Parent Directory> should have its own icon

**Describe the solution you'd like**
Something like an *out-and-up* arrow

**Describe alternatives you've considered**
N/A

**Additional context**
![parent20230501102440--](https://user-images.githubusercontent.com/9741693/235437360-b57db1e3-3431-49a2-b07a-d1109a8b95dc.png)

**Comment by IanSav** [comment]
Icons are a skinning issue.  They have nothing to do with the code.  If you want the default icons changed then I suggest you post the request there.

**Comment by jbleyel** [comment]
The icons are based on the type. Otherwise we need to change a lot to have different icons for special directories.

**Comment by IanSav** [comment]
It was never obvious and a default was not provided but the code does allow for a different icon to be used for parent directories.

The old code named the parent directory icon "back.png" but as no-one knew what it was I have just committed a change to the code to name the icon "parent.png".  Hopefully a new icon will be created for the default skin to highlight the difference for everyone.  As I suggested above, you are free to add an icon to any skins where you would like the parent directory to be highlighted.  (I suggest you name the icon "parent.png" and it will automatically activate when you get the code update.)
